src/lib/components/PointerMissBox.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
```

Comment:
Previously, deselecting the current selected item happened by attaching a `pointermiss` event handler to each object. This wasn't always reliable due to event ordering and wasn't compatible with TransformControls, which uses its own raycaster instead of Threlte's interactivity plugin.

I found it more reliable to add a big invisible box surrounding the scene to catch "deselect" clicks.